.jenkins/checks.Jenkinsfile

```diff
@@ -0,0 +1,47 @@
+pipeline {
+    libraries { lib 'adverity-shared-library@0.4.1'}
```

Comment:
@reakaleek is this version okay or is there a newer version by now that we should use?

Reply:
The latest version 2.6.. but none of the new features are used here.. up to you

Reply:
The latest version 2.6.. but none of the new features are used here.. up to you

Reply:
https://github.com/pmedianetwork/eslint-config/pull/79/commits/309753f99fe3db318e1aee1081079a1159161994